db/seeds.rb

```diff
@@ -25,14 +25,73 @@
+The painting depicts a couple embracing each other, their bodies entwined in elaborate beautiful robes decorated in a style influenced by the contemporary Art Nouveau style and the organic forms of the earlier Arts and Crafts movement. 
+The painting now hangs in the Österreichische Galerie Belvedere museum in the Belvedere, Vienna, and is considered a masterpiece of Vienna Secession (local variation of Art Nouveau) and Klimt's most popular work.",
+rate: 1092, category: "painting")
+sixth_artwork.user = User.all.sample
+
+seventh_artwork = Artwork.new(name: "Girl With a Pearl Earring", description: "Girl with a Pearl Earring (Dutch: Meisje met de parel) is an oil painting by Dutch Golden Age painter Johannes Vermeer, 
+dated c. 1665. Going by various names over the centuries, it became known by its present title towards the end of the 20th century after the earring worn by the girl portrayed there. 
+The work has been in the collection of the Mauritshuis in The Hague since 1902 and has been the subject of various literary treatments. 
+In 2006, the Dutch public selected it as the most beautiful painting in the Netherlands.", rate: 1125, category: "painting")
+seventh_artwork.user = User.all.sample
+
+eight_artwork = Artwork.new(name: "The Birth of Venus", description: "The Birth of Venus is a painting by the Italian artist Sandro Botticelli, 
+probably made in the mid 1480s. It depicts the goddess Venus arriving at the shore after her birth, when she had emerged from the sea fully-grown (called Venus Anadyomene and often depicted in art). 
+The painting is in the Uffizi Gallery in Florence, Italy.", rate: 2021, category: "painting")
+eight_artwork.user = User.all.sample
+
+ninth_artwortk = Artwork.new(name: "Las Meninas", description: "Las Meninas, Spanish for 'The Ladies-in-waiting' is a 1656 painting in the Museo del Prado in Madrid, 
+by Diego Velázquez, the leading artist of the Spanish Golden Age. Its complex and enigmatic composition raises questions about reality and illusion, 
+and creates an uncertain relationship between the viewer and the figures depicted. Because of these complexities, 
+Las Meninas has been one of the most widely analyzed works in Western painting.", rate: 1980, category: "painting")
+ninth_artwortk.user = User.all.sample
+
+tenth_artwork = Artwork.new(name: "Creation of Adam", description: "The Creation of Adam (Italian: Creazione di Adamo) is a fresco painting by Italian artist Michelangelo, 
+which forms part of the Sistine Chapel's ceiling, painted c. 1508–1512. It illustrates the Biblical creation narrative from the Book of Genesis in which God gives life to Adam, 
+the first man. The fresco is part of a complex iconographic scheme and is chronologically the fourth in the series of panels depicting episodes from Genesis.", rate: 1984, category: "painting")
+tenth_artwork.user = User.all.sample
 
 # Added 10 bookings for user Sevil
 10.times do
```

Comment:
Let's make some of these start_dates a future date so that we can show the functionality where a booking request can only be deleted if the start_date is in the future.